docusaurus/website/.env.development

```diff
@@ -0,0 +1,10 @@
+# This file is used to set environment variables for development deployments of the website.
+# DO NOT PUT SENSITIVE INFORMATION IN THIS FILE.
+DEV_PORTAL_HOST=grafana-dev.com
+DEV_PORTAL_RUDDERSTACK_URL=https://rs.grafana-dev.com
+DEV_PORTAL_RUDDERSTACK_WRITE_KEY=1w02fcWseyqcwsJA9CSKRkfEOfU
```

Comment:
Just to be totally sure, this key is not a secret?

Reply:
I don't believe it's a secret. It's a [unique id for the source](https://www.rudderstack.com/docs/resources/glossary/#write-key) of the events. It is passed through to the frontend and can be accessed via the browser network tab if someone were inclined to steal it.

![image](https://github.com/user-attachments/assets/6bca60df-5af0-4149-b1fc-04ec9233805a)